locksmith/migrations/20230425123140-key-expiration-reminder.js

```diff
@@ -39,6 +39,9 @@ module.exports = {
         type: Sequelize.DATE,
       },
     })
+    await queryInterface.changeColumn('KeyExpirationReminders', 'expiration', {
```

Comment:
You need to create a new migraion file. This one has already been executed and will not be executed again...